src/middleware.js

```diff
@@ -112,6 +114,62 @@ function errorHandler(err, req, res, next) {
   }
 }
 
+function cache(res) {
+  this.forever = (res) => {
+    const t = ms('1y');
+    res.headers.set(`Cache-Control`, `public, max-age=${t}, immutable`);
+  }; // Forever Function
+
+  /*
+no-store is needed with max-age=0
+we don't want to serve stale resources when Telescope is down.
+see: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#requiring_revalidation
+*/
+  this.never = (res) => {
+    res.headers.set('Cache-Control', 'public, no-store, max-age=0');
```

Comment:
Let's not include `public` in case it isn't. I think you can use `Cache-Control: no-store, max-age=0`